build.gradle

```diff
@@ -86,7 +86,8 @@ rat {
     'photon-ml/LINEAR_REGRESSION*/**',
     'photon-ml/LOGISTIC_REGRESSION*/**',
     'photon-ml/POISSON_REGRESSION*/**',
-    'photon-test/src/test/resources/**'
+    'photon-test/src/test/resources/**',
+    'tests.sh'
```

Comment:
Open Source department messaged me again, they've resolved their disagreements. They feel that it's best practice for build/test scripts, project management, and non-core code files in general to not have license headers. That was my line of reasoning originally, so I'm going to leave it as is.